packages/mint-components/src/components/tax-and-cash/sqm-indirect-tax-form/sqm-indirect-tax-form.feature

```diff
@@ -0,0 +1,81 @@
+@owner:andy @author:andy
+Feature: Indirect Tax Form
+
+  Background: A user has submitted their personal information in Tax Form Step One
+    Given a user is on the Indirect Tax Form
+
+  @minutia
+  Scenario Outline: Different indirect tax inputs are shown depending on the country of a participant
+    When <option> is selected based on based the participant <country> from step 1
+    Then different <inputs> appear
+
+    Examples: 
+      | country        | option                                                           | inputs                        |
+      | Canada         | I am registered for HST in Canada                                | Province, Indirect Tax Number |
+      | United States  | I am registered for Indirect Tax in a different Country / Region | Country, VAT Number           |
+      | United Kingdom | I am registered for Indirect Tax in a different Country / Region | Country, VAT Number           |
+      | Egypt          | I am not registered for Indirect tax                             | N/A, N/A                      |
+
+  @minutia @ui
+  Scenario: The province select displays applicable indirect tax provinces
+    When the "I am registered for HST in Canada" option is selected
+    Then A Province select appears with the available <provinces>
+
+    Examples: 
+      | provinces             |
+      | Ontario               |
+      | Alberta               |
+      | British Columbia      |
+      | Manitoba              |
+      | New Brunswick         |
+      | Newfoundland          |
+      | Nova Scotia           |
+      | Prince Edward Island  |
+      | Quebec                |
+      | Saskatchewan          |
+      | Northwest Territories |
+      | Nunavut               |
+      | Yukon                 |
+
+  @minutia
+  Scenario: The country select displays applicable indirect tax countries from Impacts API
+    When the "I am registered for Indirect Tax in a different Country / Region" option is selected
+    Then A Country select appears with the available countries provided by Impacts API
```

Comment:
@Locrian24 what are these countries? Its very important that the frontend only displays countries that have indirect tax that are not Canadian. Can you do some looking at their UX and see what the list is?